kale/task.py

```diff
@@ -76,6 +90,10 @@ def __init__(self, message_body=None, *args, **kwargs):
 
         self._dequeued()
 
+        if settings.is_multiprocess_mode() and not Task._task_process_pool:
```

Comment:
Maybe we should call this `master mode` ala uwsgi.  Multiprocessing mode suggests to me that we're interested in parallelism in some way.